modules/programs/docker-cli.nix

```diff
@@ -21,18 +21,26 @@ in
   options.programs.docker-cli = {
     enable = mkEnableOption "management of docker client config";
 
-    configPath = mkOption {
+    configDir = mkOption {
       type = lib.types.str;
-      default = ".docker/config.json";
+      default = ".docker";
       description = ''
-        Relative path to the user's home directory where the Docker CLI settings should be stored.
+        Folder relative to the user's home directory where the Docker CLI settings should be stored.
+      '';
+    };
+
+    configFilename = mkOption {
+      type = lib.types.str;
+      default = "config.json";
+      description = ''
+        Filename of the docker CLI settings relative to `programs.docker-cli.configDir`.
       '';
     };
```

Comment:
don't think there's really a benefit to this right? docker is still going to just search for `config.json` i thought. or does it just pick up any json files in the directory?